python/tests-unit/test_lib.py

```diff
@@ -0,0 +1,224 @@
+import pathlib
+import unittest.mock
+
+import insights_ansible_playbook_lib as lib
+import pytest
+
+
+DATA = pathlib.Path(__file__).parents[2].absolute() / "data"
+GPG_KEY = (DATA / "public.gpg").read_bytes()
+REVOKED = (DATA / "revoked_playbooks.yml").read_text()
+PLAYBOOKS = pathlib.Path(__file__).parents[2].absolute() / "data" / "playbooks"
+
+
+class TestParsePlaybook:
+    """The reference verifier used YAML 1.2.
+
+    PyYAML seems to be using YAML 1.1 by default, so we have to ensure we parse it correctly.
+    """
+
+    def test_all(self):
+        """Test that all plays loaded if present."""
+        raw = "\n".join(
+            [
+                "---",
+                "- name: first dictionary",
+                "  key: value",
+                "- name: second dictionary",
+                "  key: value",
+            ]
+        )
```

Comment:
personally speaking, a multiline block would be more readable than a list with the lines -- eg:
```python
        PLAYBOOK = """---
- name: ...
"""
```

Reply:
I dislike how they mess up indentation :(